x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorService.java

```diff
@@ -54,13 +57,22 @@ public class SlmHealthIndicatorService implements HealthIndicatorService {
         null
     );
 
-    public static final String ACTION_CHECK_RECENTLY_FAILED_SNAPSHOTS_HELP_URL = "https://ela.st/fix-recent-snapshot-failures";
-    public static final Diagnosis.Definition ACTION_CHECK_RECENTLY_FAILED_SNAPSHOTS = new Diagnosis.Definition(
-        "check_recent_snapshot_failures",
-        "The following snapshot lifecycle policies have exceeded the warning threshold for repeat failures without a successful execution.",
-        "Check the snapshot lifecycle policies [/_slm/policy/<policy_name>?human] for detailed failure info.",
-        ACTION_CHECK_RECENTLY_FAILED_SNAPSHOTS_HELP_URL
-    );
+    private static final DateFormatter FORMATTER = DateFormatter.forPattern("iso8601").withZone(ZoneOffset.UTC);
+
+    public static final String DIAGNOSIS_CHECK_RECENTLY_FAILED_SNAPSHOTS_ID = "check_recent_snapshot_failures";
+    public static final String DIAGNOSIS_CHECK_RECENTLY_FAILED_SNAPSHOTS_ACTION = "Check the snapshot lifecycle policies "
+        + "[/_slm/policy/<policy_name>?human] for detailed failure info.";
```

Comment:
Since we're on the dynamic messaging here can we have the action provide the API call with the SLM policy in it? 

ie. `/_slm/policy/test-policy?human`

Reply:
Updated code and PR description with the new messages.